cmake/stm32/utilities.cmake

```diff
@@ -11,19 +11,32 @@ function(stm32_util_create_family_targets FAMILY)
         message(FATAL_ERROR "Expected at most one core for family ${FAMILY}: ${CORES}")
     endif()
 
+    option(STM32_ENABLE_FAST_MATH "Add -ffast-math compiler flag" OFF)
+    option(STM32_NO_BUILTIN "Add -fno-builtin compiler flag" OFF)
+    option(STM32_NO_STRICT_ALIASING "Add -fno-strict-aliasing compiler flag" OFF)
+

```

Comment:
Is there a way for user to add his own (any) options ? 
If yes I would consider removing the `option` and just let him manage in his own way. I mean these options are not more important than other that the user might need.
```suggestion

```